aiven/client/cli.py

```diff
@@ -1298,6 +1298,141 @@ def service__connector__restart_task(self):
         project_name = self.get_project()
         self.client.restart_kafka_connector_task(project_name, self.args.name, self.args.connector, self.args.task)
 
+    @arg.project
+    @arg.service_name
+    @arg("schema-id", help="ID of a schema")
+    def service__schema__get(self):
+        """Get Kafka Schema Registry schema"""
+        project_name = self.get_project()
+        schema = self.client.get_schema(project_name, self.args.name, getattr(self.args, "schema-id"))
```

Comment:
No need to use `getattr` here. Just `self.args.schema_id` works.

Reply:
Positional arguments with dashes doesn't seem to be converted to underscore. Only for named arguments. I'm converting `schema_id` to named argument, so I'll remove the `getattr`.